packages/types/src/region/mutations.ts

```diff
@@ -1,24 +1,20 @@
-import { RegionCurrencyDTO } from "./common"
-
 export interface CreateRegionDTO {
   name: string
   currency_code: string
-  currency?: RegionCurrencyDTO
   countries?: string[]
-  tax_code?: string
-  tax_rate?: number
-  tax_provider_id?: string
 }
 
 export interface UpdateRegionDTO {
   id: string
   currency_code?: string
-  currency?: RegionCurrencyDTO
-  countries: string[]
+  countries?: string[]
+  name?: string
+}
+
+export interface UpdatableRegionFields {
```

Comment:
Why aren't countries updatable?, similarly, why isn't `metadata` part of `UpdateRegionDTO` and `CreateRegionDTO`?

Reply:
`metadata` should most definitly be updatable. I will add that in now. 

Updating countries in `service.update` is not supported yet. Should be tackled in a separate ticket.